tfbpmodeling/lasso_modeling.py

```diff
@@ -1474,92 +1474,140 @@ def final_model(self) -> list[str]:
         return df["selected"].tolist()
 
 
-def evaluate_interactor_significance(
+def evaluate_interactor_significance_linear(
     input_data: ModelingInputData,
     stratification_classes: np.ndarray,
     model_variables: list[str],
-    estimator: BaseEstimator = LinearRegression(fit_intercept=True),
+    estimator: BaseEstimator = LassoCV(
```

Comment:
Why is the linear estimator changed to LasosCV? This function was intended to be left the same as the original